crates/krun/src/bin/krun.rs

```diff
@@ -206,10 +207,25 @@ fn main() -> Result<()> {
         "PATH",                        // needed by `krun-guest` program
         "RUST_LOG",
     ];
+
+    // https://github.com/AsahiLinux/docs/wiki/Devices
+    const ASAHI_DEVS: [&str; 12] = [
+        "t8103", "t6000", "t6001", "t6002", "t8112", "t6020", "t6021", "t6022", "t8121", "t6030",
+        "t6031", "t6034",
+    ];
     for key in WELL_KNOWN_ENV_VARS {
         let value = match env::var(key) {
             Ok(value) => value,
             Err(VarError::NotPresent) => {
+                if key == "MESA_LOADER_DRIVER_OVERRIDE" {
+                    for compatible in
+                        read_to_string("/proc/device-tree/compatible")?.split(&[',', '\0'][..])
```

Comment:
I think we must not error here on devices that don't have `/proc/device-tree`. So just do `.ok()`

Reply:
Or rather, we can still propage other `io::Error` that are not `ErrorKind::NotFound`:
https://doc.rust-lang.org/std/io/enum.ErrorKind.html#variant.NotFound

WDYT?